models/index.js

```diff
@@ -0,0 +1,24 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+    class User extends Model {
+      static associate(models) {
+      }
+    }}
+
+    User.init(
+        {
+          username: DataTypes.STRING,
+          email: DataTypes.STRING,
+          password: DataTypes.STRING,
+        },
+        {
+          sequelize,
+          modelName: 'User',
```

Comment:
model names need to be lowercase. the model class is the only one that needs to be uppercase